CalendarApp/NUnitTestProject/BaseIntegrationTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using NUnitTestProject.Common;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+using Google.Apis.Calendar.v3.Data;
+using BookingRoom.Models.GoogleCalendar;
+using BookingRoom.Models.GoogleEvent;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public class BaseIntegrationTest
+    {
+        protected IServiceLocator _serviceLocator;
+        protected IUnityContainer _unityContainer;
+        protected BookingService _meetingBooking;
+
+        [TestFixtureSetUp]
```

Comment:
Are there reasons to initialize objects above using `TestFixtureSetUp` instead of ctor as there are no tests?